docs/options.md

```diff
@@ -192,17 +192,22 @@ Options:
 - Pyodide: `wasm32`
 - iOS: `arm64_iphoneos` `arm64_iphonesimulator` `x86_64_iphonesimulator`
 - `auto`: The default archs for your machine - see the table below.
-    - `auto64`: Just the 64-bit auto archs
-    - `auto32`: Just the 32-bit auto archs
-- `native`: the native arch of the build machine - Matches [`platform.machine()`](https://docs.python.org/3/library/platform.html#platform.machine).
+    - `auto64`: Just the 64-bit auto arch[^iosarch]
```

Comment:
This footnote ends up at the bottom of the whole page, I'll see about moving it up to be more colocated somehow...